action.yml

```diff
@@ -1,5 +1,5 @@
-name: 'Cache'
-description: 'Cache dependencies and build outputs to improve workflow execution time'
+name: 'Cache Artifacts'
```

Comment:
@chrispat Do we want to change the name? I'm worried that "Artifacts" is overloaded and the docs specifically separate the two concepts: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/caching-dependencies-to-speed-up-workflows#comparing-artifacts-and-dependency-caching

As a user, "cache artifacts", "upload artifacts", and "download artifacts" sound like they should be very similar yet "cache" is fundamentally different in use and function.

Reply:
We're not changing the name right now.

Reply:
@ethomson "Cache Artifacts" is what will show up in the Marketplace with the `action.yml` change to `name: 'Cache Artifacts'`

Reply:
Sorry, to clarify: we're not going to merge this PR as-is.  We don't want a name change.